KS.Fiks.IO.Arkiv.Client/ForenkletArkivering/ArkivmeldingFactory.cs

```diff
@@ -109,6 +114,10 @@ public static Arkivmelding GetArkivmelding(OppdaterSaksmappe input)
                     
                     var dok = new Dokumentobjekt
                     {
+                        Format = "X", //TODO Her mangler noe eller det er noe som ikke stemmer.
+                        Variantformat = "X", //TODO Her mangler noe eller det er noe som ikke stemmer.
+                        Versjonsnummer = "1", //TODO Her mangler noe eller det er noe som ikke stemmer.
+                        Filnavn = "", //TODO Her mangler noe eller det er noe som ikke stemmer.
```

Comment:
Her er det vel en mangel.

Reply:
Noark 5 angir ikke noe for Format, men Arkade 5 forutsetter filtypen, f.eks. "pdf".
Variantformat - jeg må sjekke hva vi har blitt enig om: Noark 5 sier "Produksjonsformat", "Arkivformat", GI sier "P", "A".
Versjonsnummer vil være 1 med mindre det legges ved flere versjoner av samme dokument i samme format.
Filnavn er navnet på filen slik det har vært opprinnelig. Vanlige regler. Ingen katalogangivelse.

Reply:
Blir både Filnavn og ReferanseDokumentFil det samme da? Jeg ser input kommer med atributtet `filnavn` som brukes på ReferanseDokumentFil 🤔

Reply:
Filnavn er filnavnet slik det vil være om man laster det ned.
ReferanseDokumentfil kan være en URL, full filsti etc. for henting av dokumentet. I avleveringen til arkivdepot er det sti + filnavn, i Fiks Arkiv bør det være noe man kan bruke for å hente dokumentinnholdet. Men det kan nok hende at noe av det opprinnelige formålet har blitt borte underveis.

Reply:
Det var det jeg mente også. Da putter jeg `filnavn` på `filnavn` og legger til nytt felt på input for referanseDokumentFil.